ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/DaxKeyValue.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.netflix.ndbench.plugin.dynamodb;
+
+import com.amazon.dax.client.dynamodbv2.AmazonDaxClientBuilder;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.services.dynamodbv2.model.ReturnConsumedCapacity;
+import com.google.common.base.Preconditions;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.netflix.ndbench.api.plugin.DataGenerator;
+import com.netflix.ndbench.api.plugin.NdBenchClient;
+import com.netflix.ndbench.api.plugin.annotations.NdBenchClientPlugin;
+import com.netflix.ndbench.plugin.dynamodb.configs.DaxConfiguration;
+import com.netflix.ndbench.plugin.dynamodb.operations.dynamodb.dataplane.DynamoDBReadBulk;
+import com.netflix.ndbench.plugin.dynamodb.operations.dynamodb.dataplane.DynamoDBReadSingle;
+import com.netflix.ndbench.plugin.dynamodb.operations.dynamodb.dataplane.DynamoDBWriteBulk;
+import com.netflix.ndbench.plugin.dynamodb.operations.dynamodb.dataplane.DynamoDBWriteSingle;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+@NdBenchClientPlugin("DaxKeyValue")
+public class DaxKeyValue extends DynamoDBKeyValueBase<DaxConfiguration> implements NdBenchClient {
```

Comment:
This client plugin is good. I was wondering if people would care about comparing DAX vs no DAX Side by side? If they do do we want to add some info in the documentation or add this client plugin a property that allows doing the on/off... Thoughts?

Reply:
They already can do that. Run the DaxKeyValue load on one host, and using the same configuration, run a DynamoDBKeyValue on another host. Makes sense?

Reply:
I guess we can do it in the following PR. I will merge this to get you moving forward as this PR contains a lot of material.